ui/v2/src/components/Settings/SettingsConfigurationPanel.tsx

```diff
@@ -148,6 +174,26 @@ export const SettingsConfigurationPanel: FunctionComponent<IProps> = (props: IPr
         >
           <InputGroup value={generatedPath} onChange={(e: any) => setGeneratedPath(e.target.value)} />
         </FormGroup>
+
+        <FormGroup
+          label="Excluded Patterns"
+          helperText="Regexps of files/paths to exclude from Scan"
+
+        >
+
+       { (excludes) ? excludes.map((regexp, i) => {
+                                            return(
+                                              <InputGroup
+                                              value={regexp}
+                                              onChange={(e: any) => excludeRegexChanged(i, e.target.value)}
+                                              rightElement={<Button icon="minus" intent="danger" onClick={(e: any) => excludeRemoveRegex(i)} />}
+                                              />
+                                              );
+                                            })   : null
+       }
+
+          <Button icon="plus" onClick={(e: any) => excludeAddRegex()} />
```

Comment:
I think adding `minimal={true}` to the add and remove buttons looks a bit nicer. What do you think?

Reply:
... you are probably right
normal
![normal](https://user-images.githubusercontent.com/48220860/70912523-c71d9800-201c-11ea-9795-9bd1074c5f5e.png)
minimal
![minimal](https://user-images.githubusercontent.com/48220860/70912536-cdac0f80-201c-11ea-8993-328e485f6718.png)
![minimal2](https://user-images.githubusercontent.com/48220860/70912539-d0a70000-201c-11ea-878f-99063634985b.png)

i'll set it to minimal for both